services/image-imgscalr/src/main/java/com/gentics/mesh/image/ImgscalrImageManipulator.java

```diff
@@ -265,9 +263,8 @@ private BufferedImage cropAndResize(BufferedImage image, ImageManipulationParame
 					// TODO handle execution timeout
 					// Make sure to run that code in the dedicated thread pool it may be CPU intensive for larger images and we don't want to exhaust the regular worker
 					// pool
-					Flowable<Buffer> stream = binary.getStream();
 					return workerPool.<String>rxExecuteBlocking(bh -> {
-						try (ImageInputStream ins = ImageIO.createImageInputStream(RxUtil.toInputStream(stream, vertx))) {
+						try (ImageInputStream ins = ImageIO.createImageInputStream(stream.get())) {
```

Comment:
Make sure that stream.get() inputstream gets closed as well. Wrap another try-with-resource.